src/datastore/pg-store-v2.ts

```diff
@@ -298,7 +298,7 @@ export class PgStoreV2 extends BasePgStoreModule {
         limit,
         offset,
         results: results,
-        total: results[0].total,
+        total: results[0]?.total ?? 0,
```

Comment:
throws if no cycle have been registered yet (not likely in prod, but can happen in testing)